multibase-script/multibase.py

```diff
@@ -0,0 +1,101 @@
+import os
```

Comment:
I think this folder need to be remove, since this looks repeat of utils/multibase-script